docs/core/install/windows.md

```diff
@@ -171,9 +171,9 @@ Visual Studio Code is a powerful and lightweight source code editor that runs on
 
 This [WinGet configuration file](https://builds.dotnet.microsoft.com/dotnet/install/dotnet_basic_config_docs.winget) installs the latest .NET SDK, Visual Studio Code and the C# DevKit. If you already have any of them installed, WinGet will skip that step.
 
-1. Download the file and double-click to run it.
-1. Read the license agreement, type <kbd>y</kbd>, and select <kbd>Enter</kbd> when prompted to accept.
-1. Be on the lookout for a flashing User Account Control (UAC) prompt in your Windows Taskbar, you may need to require administrator-level permissions to install.
+01. Download the file and double-click to run it.
+01. Read the license agreement, type <kbd>y</kbd>, and select <kbd>Enter</kbd> when prompted to accept.
+01. If you get a flashing User Account Control (UAC) prompt in your Taskbar, allow the installation to continue.
```

Comment:
```suggestion
1. Download the file and double-click to run it.
1. Read the license agreement, type <kbd>y</kbd>, and select <kbd>Enter</kbd> when prompted to accept.
1. If you get a flashing User Account Control (UAC) prompt in your Taskbar, allow the installation to continue.
```

Reply:
I did the 2 digit numbers to be consistent with Andy's preferred style. I'll leave this as is for now.